tests/Console/LogVerbosityTest.php

```diff
@@ -0,0 +1,100 @@
+<?php
+/**
+ * Copyright © 2017-2018 Maks Rafalko
+ *
+ * License: https://opensource.org/licenses/BSD-3-Clause New BSD License
+ */
+
+declare(strict_types=1);
+
+namespace Infection\Tests\Console;
+
+use Infection\Console\LogVerbosity;
+use Mockery;
+use Mockery\Adapter\Phpunit\MockeryTestCase;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+
+class LogVerbosityTest extends MockeryTestCase
+{
+    public function test_it_works_if_verbosity_is_valid()
+    {
+        $input = $this->set_input_expectations_when_it_does_not_change(LogVerbosity::NORMAL);
+
+        LogVerbosity::convertVerbosityLevel($input, Mockery::mock(SymfonyStyle::class));
+    }
+
+    /**
+     * @dataProvider provideConvertedLogVerbosity
+     *
+     * @param int $input
+     * @param string $output
+     */
+    public function test_it_converts_int_version_to_string_version_of_verbosity(int $input, string $output)
+    {
+        $input = $this->set_input_expectations_when_it_does_change($input, $output);
+
+        LogVerbosity::convertVerbosityLevel($input, Mockery::mock(SymfonyStyle::class));
+    }
+
+    public function provideConvertedLogVerbosity()
+    {
+        yield 'It converts none integer to none' => [
+            LogVerbosity::NONE_INTEGER,
+            LogVerbosity::NONE,
+        ];
+
+        yield 'It converts normal integer to normal' => [
+            LogVerbosity::NORMAL_INTEGER,
+            LogVerbosity::NORMAL,
+        ];
+
+        yield 'It converts debug integer to debug' => [
+            LogVerbosity::DEBUG_INTEGER,
+            LogVerbosity::DEBUG,
+        ];
+    }
+
+    public function test_it_converts_to_normal_and_writes_notice_when_invalid_verbosity()
+    {
+        $input = $this->set_input_expectations_when_it_does_change('asdf', LogVerbosity::NORMAL);
+        $io = Mockery::mock(SymfonyStyle::class);
+        $io->shouldReceive('note')
+            ->withArgs(['Running infection with an unknown log-verbosity option, falling back to \'default\' option'])
+            ->once();
+
+        LogVerbosity::convertVerbosityLevel($input, $io);
+    }
+
+    /**
+     * @param string|int $inputVerbosity
+     *
+     * @return InputInterface|Mockery\MockInterface
+     */
+    private function set_input_expectations_when_it_does_not_change($inputVerbosity)
```

Comment:
AFAIK, we use kebab_case only for test_* methods. 

* `protected function tearDown()`
* `private function setEnv(string $path)`
* `private function getPlusNode`